aries-site/src/pages/components/skeleton.mdx

```diff
@@ -0,0 +1,37 @@
+import { Notification } from 'grommet';
+import { Example } from '../../layouts';
+import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
+
+<Example
+  componentName="Skeleton"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js"
+  docs="https://v2.grommet.io/skeleton?theme=hpe#props"
+  width="large"
```

Comment:
```suggestion
template
height="500px"
```
^^ this should allow us to set a reasonable height that won't jump. See if this works.